tools/st2_deploy.sh

```diff
@@ -176,7 +176,7 @@ install_apt() {
   fi
 
   # Add StackStorm APT repo
-  echo "deb http://downloads.stackstorm.net/deb/ trusty_unstable main" > /etc/apt/sources.list.d/stackstorm.list
+  echo "deb https://downloads.stackstorm.net/deb/ trusty_unstable main" > /etc/apt/sources.list.d/stackstorm.list
```

Comment:
Could we not just add `apt-transport-https` to this file? It'll ensure we can support HTTPS, as it seems to be the big deal breaker when HTTPS with `apt` does come up.

I'll revert it, but again, another mismatch with the Puppet module, where HTTPS is default. I'd prefer HTTPS, and we encourage the transition to HTTPS everywhere versus keep legacy if we can avoid it. I'll hold judgement because the bug may be different.